src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days > 6) {
+    cost -= 50;
+  } else if (days > 2) {
+    cost -= 20;
```

Comment:
```suggestion
  if (days >= 7) {
    return cost - 50;
  }
  
  if (days >= 3) {
    return cost - 20;
```
It's better to immediately return the price to avoid using else if. 
Also, I think it's better to use 3 and 7 as it's written in task description in order not to confuse other developers

Reply:
OK, that makes sense.